package.json

```diff
@@ -158,17 +158,17 @@
     "jsdom": "^25.0.1",
     "json": "^11.0.0",
     "prettier": "^3.3.3",
-    "react": "19.0.0-rc.0",
-    "react-dom": "19.0.0-rc.0",
-    "rollup": "^4.25.0",
+    "react": "19.0.0-rc.1",
```

Comment:
Updating react to rc.1 makes tests to fail.

@kretajak I wonder if you are familiar with this case. Do we misuse RTL for async tests?

Reply:
I will take a look at that. The way RTL is used here to test async stuff is fine from my perspective. Maybe some tweaks mentioned in [React 19 upgrade guide](https://react.dev/blog/2024/04/25/react-19-upgrade-guide) are required.

Reply:
Looks like broken test are using `Suspense` and `Suspense` behavior was changed in RC1 (https://github.com/facebook/react/issues/29898#issuecomment-2477449973).

Reply:
Thanks for taking a look. I know the Suspense behavior was changed, but didn't consider if it really broke it.

https://stackblitz.com/edit/vitejs-vite-hbi7w9?file=src%2FApp.tsx&terminal=dev This looks like working. So, it's probably test only issue.